desktop/release-notes.md

```diff
@@ -24,6 +24,75 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.12.0
+2022-09-01
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+## New
+
+- Minimum version to install or update Docker Desktop on macOS is now 10.15
+- Improved overall performance issues caused by calculating disk size. Related to [docker/for-win#9401](https://github.com/docker/for-win/issues/9401).
+- Added the experimental docker daemon with the containerd image store.
+- Allow Docker Desktop to run untagged image (https://github.com/docker/for-mac/issues/6425)
+- Add search capabilities to Docker Extension Marketplace. Fixes [docker/roadmap#346](https://github.com/docker/roadmap/issues/346).
+- [mac] Quit Docker Desktop completely from the Dock menu (https://github.com/docker/for-mac/issues/6440)
+- Enable ComposeV2 after factory reset
+- Copied terminal contents do not contain non-breaking spaces anymore
+- Upgrade kernel to 5.10.124
+- Enables ComposeV2 by default on new installations/if there is no previous configuration.
+- Prevents users on ARM macs without Rosetta installed from switching back to ComposeV1 (only has intel binaries).
+- Added View menu with Zoom In, Zoom Out, and Actual Size options to control the size of the Docker Desktop UI.
+- Descending default sort order for volumes size and created column, containers started column
+- Defined sort order for container/compose state column running > some running > paused > some paused > exited > some exited > created
+- Re-organized container row actions
+- Added compose stop button if any related container is stoppable.
+- Individual compose containers are now deletable from overview
+- Runs every command on the Quickstart guide immediately.
+- Remove workaround for virtiofsd <-> qemu protocol mismatch on Fedora 35, as it is no longer needed. Fedora 35 users should upgrade the qemu package to the most recent version (qemu-6.1.0-15.fc35 as of the time of writing).
+- Implemented an integrated terminal for containers.
+- Updated the tooltip design.
```

Comment:
This should either be removed or we should explain a bit more what it means?

Reply:
Hmm, from what I can understand out of the PR, this styling change is done so we better adhere to our own UI guidelines. Perhaps a message such as this would be more relevant to users? (otherwise I agree we should just remove this entry):
Updates the tooltips in Docker Desktop and MUI theme package to align with the overall system design.

Reply:
Like your suggestion @andrei-z-albu, would just change 'Updates' to 'Updated'

Reply:
Great! I'll change it thus